### DISCUSSION TITLE
Language/Pull Request

### DISCUSSION THREAD

**Discussion by cuboooooo** [maximmax42/Discord-CustomRP#38] [Q&A]
Hey! I was wondering what language this was written in, And if it was ok if I made a pull request(I think that’s how they work) to see if I can work on a discord themed UI for the program. I’ll share it back to you when I’m done if you want!

**Comment by maximmax42** [comment]
It's written in C# with WinForms. You're more than welcome to try and design something. I've been trying for a while, particularly with a dark theme, but I can't recolor some of the stuff properly, it's frustrating. Wishing I spent some time learning WPF, it's at least more powerful.
